src/main/java/net/dries007/tfc/world/classic/fluids/FluidThirstConfig.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Work under Copyright. Licensed under the EUPL.
+ * See the project README.md and LICENSE.txt for more information.
+ */
+
+package net.dries007.tfc.world.classic.fluids;
+
+public class FluidThirstConfig
```

Comment:
Why isn't this using the already existing config? There isn't a need for a file just for what you can drink.
Oregen is an exception due to the complexity.

Reply:
This will enable support for other fluid from other mods too and would make configuration for thirst amount and potion effects later.

Reply:
The config can support that just fine...

Potion effects should be coming from the fluid, not the drinking of a jug.

Reply:
Okay, I'll change it later I guess